gpflow/optimizers/mcmc.py

```diff
@@ -21,13 +21,10 @@
 __all__ = ["SamplingHelper"]
 
 
-Parameters = Sequence[Parameter]
-LogProbabilityFunction = Callable[[], tf.Tensor]
```

Comment:
was only used once - seemed cleaner to simply write the type in the function signature (especially given how simple these are)